grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/IdFragment.java

```diff
@@ -75,4 +93,9 @@ public double fragmentCost() {
     public boolean hasFixedFragmentCost() {
         return true;
     }
+
+    @Override
+    public boolean operatesOnEdge() {
+        return id.getValue().startsWith(Schema.PREFIX_EDGE);
```

Comment:
It is javadoc'd in the interface 👍 
Will rename, though. I think the `can` makes it clearer.